components/landing/Landing.vue

```diff
@@ -4,10 +4,11 @@
       <div class="columns">
         <div class="column">
           <img
-            src="~/assets/KODADOT.png"
+            src="~/assets/KODADOT.webp"
```

Comment:
https://caniuse.com/webp

also, I see on older versions and on safari (for pre big sur versions) there is partial support for webp, would that be an issue then?

Reply:
you are absolutely right. Just added `.png` fallback for Safari pre-"Big Sur"
also fixed merge conflicts